src/main/java/net/milkbowl/vault/economy/Economy.java

```diff
@@ -23,6 +23,7 @@
 /**
  * The main economy API
  *
+ * @deprecated in lieu of the modern Vault2. To update alter your import to new.milkbowl.vault2.economy and update to use the new UUID-based methods where available.
```

Comment:
```suggestion
 * @deprecated in lieu of the modern Vault2. To update alter your import to net.milkbowl.vault2.economy and update to use the new UUID-based methods where available.
```

Reply:
Missed this in the batch of commits earlier and caught it in https://github.com/MilkBowl/VaultAPI/pull/138/commits/7c7d0abcc2a89011a5bd44f67088d6446c7cd676.